pkg/app/app.go

```diff
@@ -0,0 +1,30 @@
+package app
+
+import "github.com/giantswarm/cluster-operator/service/controller/key"
+
+var (
+	Default = key.AppSpec{
+		Catalog:         "default",
+		Namespace:       "kube-system",
+		UseUpgradeForce: true,
+	}
+
+	Exceptions = map[string]key.AppSpec{
+		"cert-exporter": {
+			Chart: "cert-exporter",
```

Comment:
Some charts kept `%s-apps` as a suffix, some don't 🤦‍♂. 
We can sort it out by adding new charts by fixing every circle CI setting, but just want to test now.